doc/UsersGuide/source/Backgroundinfo/TechnicalOverview.rst

```diff
@@ -51,7 +51,7 @@ It is maintained by EMC and DTC with developments and contributions from the :te
 
 The HAFS repository only hosts major feature branches (e.g., ``feature/hafs_nesting``) that require active group collaborations. Individual developers can work on feature branches in their personal HAFS forks. 
 
-.. figure:: https://github.com/hafs-community/HAFS/wiki/docs_images/hafs_branching_diagram.png
+.. figure:: https://raw.githubusercontent.com/wiki/hafs-community/HAFS/docs_images/hafs_branching_diagram.png
```

Comment:
@jdkublnick This will work, but it is better just to add one line to `conf.py` instead of changing all of these links because the error will continue to pop up every time someone adds a `https://github.com/hafs-community/HAFS/wiki/` link. By putting it in `conf.py` as an allowed redirect, both versions of the link will work. I've suggested a code change in `conf.py` and recommend keeping these as they were for uniformity with other UFS code, which does not use the "raw" links.